api/src/main/java/org/openmrs/module/patientqueueing/model/PatientQueue.java

```diff
@@ -67,6 +68,12 @@
 	@JoinColumn(name = "queue_room")
 	private Location queueRoom;
 	
+	@Column(name = "date_picked")
```

Comment:
Can you describe what this field is to be used for?

Reply:
@dkayiwa  this helps differentiate between service time and waiting time.
When a patient is picked  by a provider there is need to know when they were picked from the queue to be served. This works best when patients a queued to a pool without specifying clinician. 
Date time Picked is the same as Date when patient entered into the clinician room.

Reply:
@dkayiwa  done

Reply:
Do you mind fixing the grammar?